src/utils/EnvironmentUtils.js

```diff
@@ -0,0 +1,41 @@
+import {Storage} from '../libs/storage';
+import {includes} from 'lodash/fp';
+
+
+/**
+ * Predefined groups of environments for which certain features might be valid for.
+ * The environment hierarchy is:
+ *  * prod
+ *    * staging
+ *      * alpha
+ *      * dev
+ *      * local
+ * @type {{NON_STAGING: string[], NON_PROD: string[]}}
+ */
+export const envGroups = {
+  NON_PROD: ['local', 'dev', 'alpha', 'staging'],
+  NON_STAGING: ['local', 'dev', 'alpha']
```

Comment:
I imagine these environments will be pretty stable, but I wonder if it would be useful to be extensible somehow:

```
const ALL_ENVS = [ 'local', 'dev', 'alpha', 'staging', 'prod' ];
const NON_PROD = _.without(ALL_ENVS, 'prod');
const NON_STAGING = _.without(ALL_ENVS, 'staging', 'prod');

export const envGroups = {
    ALL_ENVS: ALL_ENVS,
    NON_PROD: NON_PROD,
    NON_STAGING: NON_STAGING
};
```

Perhaps this is too much unnecessary complexity though.

Reply:
I think this would be a good idea if we needed something else that used `ALL_ENVS`. I think what I have now is extensible enough for future development and yet is the minimum needed to accomplish the goal.